backend/server.js

```diff
@@ -2,6 +2,21 @@ const express = require("express");
 const app = express();
 const router = require("./route/router");
 
+const events = require("./data/events.json");
+const lodash = require("lodash");
```

Comment:
I don't see where these dependencies are used? If they are not needed can you remove them?